src/components/sequencing/form/ExtraArgumentEditor.svelte

```diff
@@ -6,7 +6,7 @@
   export let setInEditor: () => void;
 </script>
 
-<div>
-  <button on:click={setInEditor}>Remove</button>
-  <span>{initVal}</span>
+<div style="display: flex; gap: 8px">
```

Comment:
Since this is a static style, can this be moved into `<style></style>` tags, please?